iree/samples/static_library/static_library_demo.c

```diff
@@ -82,11 +82,6 @@ iree_status_t Run() {
   if (iree_status_is_ok(status)) {
     status = create_device_with_static_loader(&device);
   }
-  iree_vm_module_t* hal_module = NULL;
-  if (iree_status_is_ok(status)) {
-    status =
-        iree_hal_module_create(device, iree_allocator_system(), &hal_module);
```

Comment:
(redundant since the `iree_runtime_session_create_with_device` call below does this already)